getting-started/setup-multi-node-advanced.md

```diff
@@ -0,0 +1,370 @@
+
+##### 1. Set up a Certification Authority
+
+A Certificate Authority (CA) is necessary as a trusted third party to 
+sign other certificates.  The _key_ of the CA is used to sign 
+Certificate Signing Requests (CSRs), and the _certificate_ of the CA is 
+what is used as a root certificate for other parties. Creating a new CA 
+is not necessary if there is already one available to be employed.  In 
+that case skip to the next step.
+
+>:WARNING: The Certificate Authority key is central to your security
+and needs to be treated with care. Ideally, the key should be
+generated and used on a secured machine dedicated for this
+purpose. Typically, the key should not leave the machine and you
+should even consider the risk of backups being compromised.
+
+First, generate a key and insert it into your `auth.key` file:
+
+```bash
+openssl genpkey -algorithm rsa -out auth.key
+```
+
+Generate a self-signed root certificate for the CA:
+```
+openssl req -new -key auth.key -days 3650 -out root.crt -x509
+
+You are about to be asked to enter information that will be incorporated
+into your certificate request.
+What you are about to enter is what is called a Distinguished Name or a DN.
+There are quite a few fields but you can leave some blank
+For some fields there will be a default value,
+If you enter '.', the field will be left blank.
+-----
+Country Name (2 letter code) [AU]:US
+State or Province Name (full name) [Some-State]:New York
+Locality Name (eg, city) []:New York
+Organization Name (eg, company) [Internet Widgits Pty Ltd]:Example Company Pty Ltd
+Organizational Unit Name (eg, section) []:
+Common Name (e.g. server FQDN or YOUR name) []:http://cert.example.com/
+Email Address []: 
+```
+
+`Common Name` is the (imaginary) location of the root
+certificate, which in theory would allow a user to download the
+certificate from a known location and verify that it is the correct
+one. In this case, the value is not checked but it is possible to have
+different policies on what is allowed as common name.
+
+Remember that the root certificate only contains public information and
+can be distributed freely between machines. It is actually more secure
+to distribute it widely so that anyone interested in using the root
+certificate can be more certain to get the right certificate if it
+is available at many locations.
+
+##### 2. Generate keys and certificates for instances
+
+Keys and certificates serve similar but distinct purposes for the data nodes and 
+access node respectively.  For the data nodes, a signed certificate verifies the 
+node to the access node.  The access node needs a signed certificate to sign user 
+certificates for access.
+
+The default names for the instance key and certificate are `server.key`
+and `server.crt` respectively and they are both placed in the data
+directory of the instance. To create a server certificate:
+
+1. Generate a Certificate Signing Request `server.csr` for the
```

Comment:
I think we must mention that the Common Name field should be the host name or IP address of the server/node.